schemas/2014-04-01/Microsoft.Sql.json

```diff
@@ -84,6 +87,52 @@
       ],
       "description": "Microsoft.Sql/servers"
     },
+    "servers_administrators": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
```

Comment:
It shouldn't hurt anything. Weird tho' ... I'm not sure why it'd generate that. 

I filed a bug, but I can't fix it right yet... https://github.com/Azure/autorest/issues/2401